### DISCUSSION TITLE
Is it possible to add "cutmix" augmentation??

### DISCUSSION THREAD

**Discussion by leesangjoon1** [facebookresearch/detectron2#3286] [Q&A]
I want to add other augmentations 

So I wonder it is possible to add other augmentation like "cutmix"

and if it is possible, How do I do that?? 

Thank you!!

**Comment by snakeztc** [comment]
Same request here. Also mosaic too

**Comment by zhanghang1989** [comment]
It might be quite complicated to achieve this in detectron2. Since it can't be achieved in data mapper, modifying collate function is needed.
